styles.css

```
@@ -0,0 +1,220 @@
+* {
+  box-sizing: border-box;
+}
+/* variables*/
+:root {
+  --color-blue: #1d3875;
+  --color-box: hsl(223, 54%, 97%);
+  --color-highlight: #c2acfa;
+  --color-pink: #ff759c;
+
+  --transition-default: all 200ms ease;
+}
+
+/* UTILITY STYLES*/
+.u-list-inline li {
+  display: inline-block;
+  font-family: "Archivo Black", sans-serif;
+}
+.u-list-reset {
+  list-style: none;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-left: 0;
+}
+
+body {
+  background-color: #11182b;
+  color: #403f3f;
+  font-family: "Open Sans", sans-serif;
+  font-size: 18px;
+  margin: 0 30px;
+  text-align: left;
+}
+
+button {
+  background-color: var(--color-pink);
+  border: none;
+  border-radius: 15px;
+  color: #1d3875;
+  font-weight: 800;
+  font-size: 20px;
+  padding: 17px 45px;
+}
+h1 {
+  color: white;
+  font-family: "Archivo Black", sans-serif;
+  font-size: 50px;
+  line-height: 60px;
+}
+h2 {
+  font-family: "Archivo Black", sans-serif;
+  color: var(--color-blue);
+  font-size: 40px;
+  line-height: 120px;
+  text-align: center;
+  text-transform: uppercase;
+}
+h3 {
+  color: #427ec7;
+  font-size: 35px;
+  font-weight: 600;
+  line-height: 30px;
+}
+header {
+  background-color: var(--color-blue);
+  margin: 0 -30px;
+  padding: 30px;
+}
+main {
+  background-color: var(--color-box);
+  padding: 24px;
+}
+nav {
+  display: block;
+  margin: 0;
+  text-align: right;
+}
+
+form li {
+  margin-top: 1em;
+  padding: 3px;
+}
+
+form button {
+  border-radius: 10px;
+  margin-left: 105px;
+  padding: 9px 23px;
+}
+
+label {
+  display: inline-block;
+  text-align: right;
+  width: 100px;
+}
+
+footer {
+  color: white;
+  padding: 20px 0;
+}
+
+img {
+  max-height: 400px;
+  object-fit: cover;
+  position: relative;
+  width: 350px;
+}
+
+ul {
+  font-size: 25px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.button-styled {
+  color: white;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+  width: 60px;
+}
+
+.important-text {
+  font-size: 25px;
+  font-weight: 600;
+  line-height: 40px;
+}
+.important-text em {
+  color: var(--color-pink);
+  font-weight: 800;
+}
+/* hover styles*/
+nav a:hover {
+  background-color: #122252;
+  color: var(--color-highlight);
+  transition: var(--transition-default);
+}
+
+button:hover {
+  background-color: #852c55;
+  color: var(--color-highlight);
+  transition: var(--transition-default);
+}
+/* media inquiries */
+
+@media (max-width: 550px) {
+  .site-nav > button:not([aria-expanded]) {
+    display: none;
+  }
+  .site-nav > button[aria-expanded="false"] + ul {
+    display: none;
+  }
+  .site-nav {
+    position: absolute;
+    top: 0;
+    right: 0;
+
+    z-index: 1;
+  }
+
+  .menu-list {
+    animation: slideIn 0.5s forwards;
+    background-color: #122252;
+    display: block;
+    padding-top: 20px;
+    right: 20px * -1;
+    top: 13px * -1;
```

I did this calculation in my example because I was using variables; here, since you're using specific numbers, you can just update them to be negative numbers without the extra complexity:
```suggestion
    right: -20px;
    top: -13px;
```

---

Also, since this list doesn't have an updated `position` property, these two rules aren't actually doing anything and can just be deleted.

The same is also true for the `display` and `z-index` properties here. If you remove them, the menu still looks and functions as expected.